src/fu-device-metadata.h

```diff
@@ -22,8 +22,30 @@
 #ifndef __FU_DEVICE_METADATA_H__
 #define __FU_DEVICE_METADATA_H__
 
-/* thunderbolt plugin */
+/**
+ * SECTION:fu-device-metadata
+ * @short_description: a device helper object
+ *
+ * An object that makes it easy to close a device when an object goes out of
+ * scope.
+ *
+ * See also: #FuDevice
+ */
+
+/**
+ * FU_DEVICE_METADATA_TBT_CAN_FORCE_POWER:
+ *
+ * If the hardware can force-enable the Thunderbolt hardware.
```

Comment:
Maybe instead "If the system can force-enable the Thunderbolt controller"

Reply:
Fixed, thanks.